tensorboard/plugins/hparams/summary_v2.py

```diff
@@ -197,10 +200,29 @@ def _normalize_hparams(hparams):
             k = k.name
         if k in result:
             raise ValueError("multiple values specified for hparam %r" % (k,))
-        result[k] = v
+        result[k] = _normalize_numpy_value(v)
     return result
 
 
+def _normalize_numpy_value(value):
+    """Convert a Python or Numpy scalar to a Python scalar.
```

Comment:
Is there a non-numpy scalar data type? If so, how does this method behave?

Reply:
Sure: I’d call `tf.constant(1.0)` a non-Numpy scalar data type. The API
of the method is that behavior on that input is undefined, because it
doesn’t meet the `Args:` requirement. The actual behavior is that it’s
passed through unchanged.

Does this sound reasonable, or would you like to see additional docs or
implementation changes?